### ISSUE TITLE
create testdata

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#137]
** Prerequisite*** Can only be done after #134 

This should be done by everyone. And as much as you find time.

## User Story
1. As a developer
2. I need sufficient test data in the correct structure
3. So that I can train an AI model

## Acceptance criteria
- [ ] data includes appropriate use cases
- [ ] data follows the proposed structure 

## Definition of done (DoD)
- Work products are uploaded to the Github repository and merged to the main branch
- If necessary, the bill of materials section of the planning documents is updated
- Code/feature passes suited tests
- Code is peer-reviewed
- Help documentation (README) is updated
- Fits into our architecture, where possible

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
